### PR TITLE
Indoor directions/init mb1

### PR DESCRIPTION
**Pull request by claudiiafg**
- Indoor navigation for 1st floor of JMSB building
- Connecting outdoor pages to indoor pages
- Major refactoring in overall files organizations
- Search path algorithm

Going from outside page to inside (by clicking on a specific building):
Captura de ecrã 2020-02-29, às 9 25 10 PM

Floor plan
Captura de ecrã 2020-02-29, às 9 25 20 PM

Path found
Captura de ecrã 2020-02-29, às 9 25 54 PM

### CODE REVIEW

**Comment by claudiiafg** [comment]
PullRequest: SucceededWithIssues

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
-import { Component } from '@angular/core';
+import { Component } from "@angular/core";
```
Comment (ds161833) [inline src/app/app.component.ts]: Dont we want these as singular? I dont care tho, as long as they wont cause merge issues for others

**Comment by claudiiafg** [comment]
my prettier is not working as expected, someone will have to do that

**Comment by claudiiafg** [comment]
Azure DevOps CI pipeline automated comment:

PR Number: 129

PullRequest-381982647: SucceededWithIssues

**Comment by claudiiafg** [comment]
Azure DevOps CI pipeline automated comment:

PR Number: 129

PullRequest-381982647: SucceededWithIssues

**Pull request merged**